huntsman/utils/light_control.py

```diff
@@ -0,0 +1,265 @@
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username
+
+#note, light 5 can be 'uncommented' once installed, or causes problems using currently 
+
+
+#deal with error system - should errors be sent to pocs? how does pocs handle error messages?
+
+def manual_login_phillips_hue(hue_ip="10.88.21.10", verbose = False):
```

Comment:
I would rename file to `hue_light_control.py` and then drop the `phillips_hue` from all the function names.

Reply:
Fixed